tests/integration/linux_bridge_test.py

```diff
@@ -158,6 +158,14 @@ def test_create_vlan_as_slave_of_linux_bridge(port0_vlan101):
         assertlib.assert_state(desired_state)
 
 
+def test_create_vlan_over_linux_bridge(bridge0_with_port0):
+    vlan_base_iface = bridge0_with_port0[Interface.KEY][0][Interface.NAME]
```

Comment:
`bridge0_with_port0` contains two interfaces (the bridge and the port), how do you know who is at index 0?

Reply:
Are you okay with me hard coding the `vlan_base_iface` then? 
or would you like to give any recommendations?